reporter/src/main/kotlin/utils/FreemarkerTemplateProcessor.kt

```diff
@@ -199,6 +199,11 @@ class FreemarkerTemplateProcessor(
          */
         val licenseFiles: ResolvedLicenseFileInfo by lazy { input.licenseInfoResolver.resolveLicenseFiles(id) }
 
+        /**
+         * A name of this package that can be referenced from templates. It is generated from the identifier.
+         */
+        val name: String = "${id.namespacePrefix()}${id.name}:${id.version}"
```

Comment:
Can't we rather make use of `Identifier.toCoordinates()` or `Identifier.toPath()` here (maybe after generalizing their implementation), instead of coming up with something completely new?

Reply:
I dropped this change, also because of the comment of @fviernau.